Makefile

```diff
@@ -105,7 +105,7 @@ k8s-e2e: ## Run k8s specific e2e test
 		-ssh-key $${SSH_PK:-~/.ssh/id_rsa} \
 		-machine-controller-image $${ACTUATOR_IMAGE:-gcr.io/k8s-cluster-api/aws-machine-controller:0.0.1} \
 		-machine-manager-image $${ACTUATOR_IMAGE:-gcr.io/k8s-cluster-api/aws-machine-controller:0.0.1} \
-		-nodelink-controller-image $$(docker run registry.svc.ci.openshift.org/openshift/origin-release:v4.0 image machine-api-operator) \
+		-nodelink-controller-image $$(docker run registry.svc.ci.openshift.org/origin/release:4.2 image machine-api-operator) \
```

Comment:
Can this be a separate PR?w

Reply:
this needs to be updated here as it contains the mao changes to make tests pass with the vendored cluster-api (no cluster api node controller)